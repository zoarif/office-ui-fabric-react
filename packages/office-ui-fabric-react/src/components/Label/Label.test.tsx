import * as React from 'react';

import * as ReactDOM from 'react-dom';
import * as ReactTestUtils from 'react-dom/test-utils';
import * as renderer from 'react-test-renderer';

import { Label } from './index';

describe('Label', () => {

  it('renders a label', () => {
    const component = ReactTestUtils.renderIntoDocument(
      <Label>test</Label>
    );
    const renderedDOM = ReactDOM.findDOMNode(component as React.ReactInstance) as Element;

    expect(renderedDOM.textContent).toEqual('test');
  });

  it('renders label correctly', () => {
    const component = renderer.create(
      <Label>test</Label>
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

});
