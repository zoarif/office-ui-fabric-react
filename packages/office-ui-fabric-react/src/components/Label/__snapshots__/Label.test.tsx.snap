// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label renders label correctly 1`] = `
<label
  className=
      ms-Label
      {
        box-shadow: none;
        box-sizing: border-box;
        color: #333333;
        display: block;
        margin-bottom: 0px;
        margin-left: 0px;
        margin-right: 0px;
        margin-top: 0px;
        overflow-wrap: break-word;
        padding-bottom: 5px;
        padding-left: 0;
        padding-right: 0;
        padding-top: 5px;
        word-wrap: break-word;
      }
>
  test
</label>
`;
